### DISCUSSION TITLE
Data deklaracji czasu wysyłki

### DISCUSSION THREAD

**Discussion by bseHlpDsk** [allegro/allegro-api#5740] [Suggestions - resolved]
W momencie zwiększania (słusznie) nacisku na czas realizacji zamówień - niezbędne byłoby umożliwienie pobierania przez API daty spodziewanej wysyłki. W zasadzie w metodzie 

https://api.allegro.pl/order/checkout-forms

jest takie pole (delivery->time->guaranteed->to) 
ale jego przeznaczenie jest inne (Guaranteed date when first delivery attempt takes place. This is always filled for X-press Couriers delivery method.) 

Czy jest jakaś inne metoda?

**Comment by PrzemyslawLukanowski** [comment]
Na tę chwilę nie zwracamy w API takiego pola. Jeśli jednak zdecydujemy się je wprowadzić, poinformujemy o tym na naszej stronie oraz forum.

**Comment by MartaNowaczyk** [comment]
Wdrożyliśmy [zmiany w zamówieniach](https://developer.allegro.pl/news/zamowienia-wprowadzimy-czas-dostawy-i-wysylki-wwemeokPzF1), które powinny odpowiadać twoim oczekiwaniom.
